pyoverkiz/client.py

```diff
@@ -122,6 +126,10 @@ def __init__(
 
         self.session = session if session else ClientSession()
 
+        if "/enduser-mobile-web/1/enduserAPI/" in server.endpoint:
```

Comment:
Would be better to add a property named for instance. `support_local`. WDYT?

Reply:
I was in doubt about this one as well... In the end I would love to see something like this (https://github.com/iMicknl/python-overkiz-api/issues/321), where we define the auth mechanism in the `OverkizServer` definition. 

Eventually that would be even better, since we can than easily pick any authentication method, and we don't need an extra property just for local.

Reply:
I simplified it in cfa2abc08e52eae553b25a4d369f0c720f8b1bdc, would that be ok for now? @tetienne